src/Project/ToolRunner.ts

```diff
@@ -83,12 +84,10 @@ export class ToolRunner {
     return oneccRealPath;
   }
 
-  public getRunner(name: string, onecc: string, tool: string, toolargs: ToolArgs, path: string) {
+  public getRunner(job: Job, path: string) {
```

Comment:
In `JobRunner.ts`, it is reasonable to change previous code to getRunner(job, path) but
in `ToolRunner.ts`, I am not sure this needs to take `job` as parameter. 
IMHO, I guess `ToolRunner` handles level lower underneath `JobRunner`, and `ToolRunner` is used by `JobRunner` or some other classes which want to run some tool with string params.

Reply:
> I guess ToolRunner handles level lower underneath JobRunner, and ToolRunner is used by JobRunner or some other classes which want to run some tool with string params.

That's very reasonable. I'll fix it. And FYI the reason why I changed is to ease make it call the callback function.

Reply:
Now I see that fixing `toolrunner` is not necessary. Thanks!